overlays/rust.nix

```diff
@@ -0,0 +1,12 @@
+self: super:
+let
+  rustChannel = self.rustChannelOf { date = "2020-02-04"; channel = "nightly"; };
```

Comment:
Do they have stable releases, or do you just pick a nightly to use?

Reply:
I just picked the earliest nightly I could. Naersk uses an unstable flag that is only available in nightly rust builds.